<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OVRImageQualityClass</class>
 <widget class="QMainWindow" name="OVRImageQualityClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distortion Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuDistortion_Shader">
     <property name="title">
      <string>Distortion Shader</string>
     </property>
     <addaction name="actionNone"/>
     <addaction name="actionStandard"/>
    </widget>
    <widget class="QMenu" name="menuFiltering_Mode">
     <property name="title">
      <string>Filtering Mode</string>
     </property>
     <addaction name="actionGL_NEAREST"/>
     <addaction name="actionGL_LINEAR"/>
    </widget>
    <widget class="QMenu" name="menuTest_Pattern">
     <property name="title">
      <string>Test Pattern</string>
     </property>
     <addaction name="actionPixel_Grid"/>
     <addaction name="actionGradient"/>
    </widget>
    <addaction name="menuDistortion_Shader"/>
    <addaction name="menuFiltering_Mode"/>
    <addaction name="menuTest_Pattern"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="StatusWidget" name="statusBar"/>
  <action name="actionNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionGL_NEAREST">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>GL_NEAREST</string>
   </property>
  </action>
  <action name="actionGL_LINEAR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GL_LINEAR</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPixel_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixel Grid</string>
   </property>
  </action>
  <action name="actionGradient">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StatusWidget</class>
   <extends>QStatusBar</extends>
   <header>StatusWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="distortiontest.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>OVRImageQualityClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>OVRImageQualityClass</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>about()</slot>
 </slots>
</ui>
